wp-content/themes/access/src/js/screener.js

```diff
@@ -19,4 +24,38 @@ import Tooltips from 'utilities/tooltips/tooltips';
   (elements => {
     elements.forEach(element => new Tooltips(element));
   })(document.querySelectorAll(Tooltips.selector));
+
+  /**
+   * Initialize the Share Form and Disclaimer
+   */
+  (elements => {
+    elements.forEach(element => {
+      let shareForm = new ShareForm(element);
+      let strings = Object.fromEntries([
+        'SHARE_FORM_SERVER',
+        'SHARE_FORM_SERVER_TEL_INVALID',
+        'SHARE_FORM_VALID_REQUIRED',
+        'SHARE_FORM_VALID_EMAIL_INVALID',
+        'SHARE_FORM_VALID_TEL_INVALID'
+      ].map(i => [
+        i.replace('SHARE_FORM_', ''),
+        localize(i)
+      ]));
+
+      shareForm.strings = strings;
+      shareForm.form.strings = strings;
+
+      shareForm.sent = instance => {
+        let key = instance.type.charAt(0).toUpperCase() +
+          instance.type.slice(1);
+
+        Track.event(key, [
```

Comment:
It looks like there's a dependency on the tracking function here that will also need to be imported. Also, can we dry this a little bit by adding this block to a /module script and simply import it once? 

Thanks @theasteve

Reply:
@devowhippit updated PR with module script.